### PR TITLE
docs: Clean up quick start steps for editing the task

### LINKED ISSUE #94: "Quick Start" step for creating a new file
The user's `build__Timeline` function should be in a new file.

- [ ] Create the new file and add a trial (hello world)
- [ ] Update the import of `buildHoneycombTimeline` to use the new timeline

### LINKED ISSUE #95: "Quick Start" -> "Run the task in development mode" needs updating

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
Updates the quick start guide! Lots more screenshots I do apologize.

- Adds 3steps under "Edit the Task" 
- Adds a separate step for creating a new file for updating the `experiment/index.js` file
- Adds a separate step for adding the `startProcedure` and `endProcedure` to the new task
- Adds a magic comment that highlight's lines in  a code block red (tells the user to delete that line)

image
image
image
image
image

### CODE REVIEW

File: docs/quick_start.mdx [hunk 1]
```diff
@@ -221,22 +221,30 @@ npm install -g electron firebase-tools dotenv-cli
...
 
-Running the task in development mode causes it to hot-reload whenever changes are made to the app. This is how you'll run the project while building your task.
-
-```shell title="Run the task in dev mode"
+```shell title="Run the task in development mode"
 npm run dev
 ```
 
-This script launches an Electron window with the task and inspector open.
+Running the task in development mode causes allows for hot-reloading whenever changes are made to the app. This is how you'll run the project while creating your task!
```
Comment (galenwinsor) [inline docs/quick_start.mdx]:
```suggestion
Running the task in development mode enables "hot reloading": when you make changes to the code, they'll immediately be reflected in the app without you needing to restart the server. This is how you'll run the project while creating your task!
```

File: docs/quick_start.mdx [hunk 2]
```diff
@@ -221,22 +221,30 @@ npm install -g electron firebase-tools dotenv-cli
...
+:::
 
 ## Edit the Task
 
-Now that the task is up and running we can make our first changes to the code! We'll edit the `package.json` file to reflect your information.
+Now that the project is up and running we can make our first changes to the code!
+
+### 1) Edit the Project Metadata
 
 1. Create a new branch
```
Comment (galenwinsor) [inline docs/quick_start.mdx]: Do you need to specify that this command is run in the Terminal and without killing the server?

Comment (galenwinsor) [reply docs/quick_start.mdx]: I mean, you could rerun it, but maybe folks need to be reminded that the server needs to be running constantly or else the app will close?

File: docs/quick_start.mdx [hunk 3]
```diff
@@ -245,16 +253,256 @@ Now that the task is up and running we can make our first changes to the code! W
...
+     console.log(`Building timeline for participant ${participantID} on study ${studyID}`);
+
+     // highlight-delete-next-line
+     const timeline = buildHoneycombTimeline(jsPsych);
+     const timeline = buildTaskNameTimeline(jsPsych);
+     return timeline;
+   }
+   ```
+
+9. Run a format to make sure the code is formatted correctly
```
Comment (galenwinsor) [inline docs/quick_start.mdx]:
```suggestion
9. Run the format script to make sure the code is formatted correctly
```

File: docs/quick_start.mdx [hunk 4]
```diff
@@ -245,16 +253,256 @@ Now that the task is up and running we can make our first changes to the code! W
...
+   ```
+
+9. Save your changes and commit them to git:
+
+   ```shell
+   git commit -a -m "feat: adds endProcedure to the task"
    ```
 
-4. Create a [pull request](version_control#create-a-pull-request) to bring your changes into the `main` branch
+10. Create and merge a [pull request](version_control#create-a-pull-request) to merge your changes into the `main` branch. Make sure the builds complete successfully before merging!
```
Comment (galenwinsor) [inline docs/quick_start.mdx:505]: Do you want to give instructions on what it means to make sure the build completes successfully?

**Review by galenwinsor — CHANGES_REQUESTED** [review]
Looks great! Just some nitpicky changes to language and questions about clarity.

File: docs/quick_start.mdx (hunk 2 above)
Comment (RobertGemmaJr) [reply docs/quick_start.mdx]: I think it's okay to leave it as is. Either closing the server and changing branches or changing branches in a new terminal will work.  We document that the `npm run dev` will reflect the changes immediately so whether or not they keep it running is up to them

File: docs/quick_start.mdx (hunk 4 above)
Comment (RobertGemmaJr) [reply docs/quick_start.mdx:505]: The builds are run as github actions and I think GitHub makes it pretty self explanatory if/when the actions fail and how to diagnose the issues. I'm toying around with the idea of writing a seperate GitHub Discussions post for working with the actions but I don't think that needs to be up before 3.3 gets released

File: docs/quick_start.mdx [hunk 5]
```diff
@@ -221,22 +221,30 @@ npm install -g electron firebase-tools dotenv-cli
...
 
-Running the task in development mode causes it to hot-reload whenever changes are made to the app. This is how you'll run the project while building your task.
-
-```shell title="Run the task in dev mode"
+```shell title="Run the task in development mode"
 npm run dev
 ```
 
-This script launches an Electron window with the task and inspector open.
+Running the task in development mode enables "hot reloading": when you make changes to the code, they'll immediately be reflected in the app without you needing to restart the server. This is how you'll run the project while creating your task!
```
Comment (RobertGemmaJr) [inline docs/quick_start.mdx]:
```suggestion
Running the task in development mode enables "hot reloading": changes to the code will immediately be reflected in the app without needing to restart the server. This is how you'll run the project while creating your task!
```

File: docs/quick_start.mdx (hunk 2 above)
Comment (RobertGemmaJr) [reply docs/quick_start.mdx]: I updated the text to say "Create a new branch in a separate terminal" which should help a little

File: docs/quick_start.mdx [hunk 6]
```diff
@@ -221,22 +221,30 @@ npm install -g electron firebase-tools dotenv-cli
...
+:::
 
 ## Edit the Task
 
-Now that the task is up and running we can make our first changes to the code! We'll edit the `package.json` file to reflect your information.
+Now that the project is up and running we can make our first changes to the code!
+
+### 1) Edit the Project Metadata
 
 1. Create a new branch
```
Comment (RobertGemmaJr) [inline docs/quick_start.mdx]:
```suggestion
1. Create a new branch in a separate terminal
```

**Review by galenwinsor — APPROVED** [review]

**Pull request merged**